planning_domains_api.py

```diff
@@ -68,7 +76,7 @@ def update_stat(stat_type, iid, attribute, value, description):
               'value': value,
               'desc': description}
 
-    res = query("/classical/update%s/%d" % (stat_type, iid),
+    res = query("/update%s/%d" % (stat_type, iid),
```

Comment:
Woudn't `query` need the formalism as well?

Reply:
Since the formalism is defaulted to "classical", if I don't pass anything in for formalism it should still work I believe

Reply:
But if we don't use the default, we'll want to pass on that new selection.

Reply:
![image](https://user-images.githubusercontent.com/71029388/231926104-9ca8dd35-2dbf-41d4-8705-a24e07a4a95f.png)
Do I have to do the same for this get_collections() function?

Reply:
Yep! We'll now have collections from various formalisms now.